posthog/settings/sentry.py

```diff
@@ -43,6 +43,8 @@ def traces_sampler(sampling_context: dict) -> float:
             return 0.00001  # 0.001%
         elif path.startswith("/api"):
             return 0.001  # 0.1%
+        elif path.startswith("/api/projects") and ("dashboard" in path or "insight" in path):
```

Comment:
This needs to be above the condition on line 44?

Reply:
right :sweat_smile: